packages/core/src/browser/storage-service.ts

```diff
@@ -76,4 +84,31 @@ export class LocalStorageService implements StorageService {
         const pathname = typeof window === 'undefined' ? '' : window.location.pathname;
         return `theia:${pathname}:${key}`;
     }
+
+    private ShowMessageNotification() {
```

Comment:
Small typo with the use of incorrect `PascalCase` method name. Also, perhaps rename the method to something less generic such as `showDiskQuotaExceededMessage`/`showStorageExceededMessage` or along similar lines.

Reply:
Done , use showDiskQuotaExceededMessage